src/libraries/System.Private.CoreLib/src/System/Activator.cs

```diff
@@ -40,12 +41,18 @@ public static partial class Activator
         public static object? CreateInstance(Type type) =>
             CreateInstance(type, nonPublic: false);
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2006:UnrecognizedReflectionPattern",
```

Comment:
Shouldn't these 3 be `RequiresUnreferencedCode`?

Reply:
Just to confirm, we will delete the all the `string,string` handling from linker and mark all these as `RequiresUnreferenced`?

Reply:
I think we should make the two independent. The annotations should be correct regardless of intrinsics in the linker.

I'm fine removing these intrinsics from the linker as well BTW.

Reply:
I didn't realize intrinsic handling would trump the RequiresUnreferenced.

In that case, RequiresUnreferenced makes sense here.